examples/with-tailwind/apps/web/postcss.config.js

```diff
@@ -7,7 +7,17 @@ module.exports = {
   plugins: {
     // Specifying the config is not necessary in most cases, but it is included
     // here to share the same config across the entire monorepo
-    tailwindcss: { config },
+    tailwindcss: {
+      config: {
+        ...config,
+        content: [
+          "src/**/*.{js,ts,jsx,tsx}",
+          // Including ui's components so the tailwind can generate classes
+          // along with hosted (apps/web) app, and prevent classes duplicates
+          "../../packages/ui/dist/**/*.{js,ts,jsx,tsx}",
```

Comment:
While this does work just fine, we wanted to avoid this option in the example to prevent promoting breaking monorepo app boundaries. However, we do already recommend this approach as an option in the README for this example ([here](https://github.com/vercel/turbo/tree/main/examples/with-tailwind#building-packagesui)):

> This example is setup to build packages/ui and output the transpiled source and compiled styles to dist/. This was chosen to make sharing one tailwind.config.js as easy as possible, and to ensure only the CSS that is used by the current application and its dependencies is generated.
Another option is to consume packages/ui directly from source without building. If using this option, you will need to update your tailwind.config.js to be aware of your package locations, so it can find all usages of the tailwindcss class names.

Reply:
@tknickman - Thanks for your explanation. 👍